libs/testserver/fake_workspace.go

```diff
@@ -178,7 +179,14 @@ func (s *FakeWorkspace) WorkspaceFilesImportFile(filePath string, body []byte) {
 
 	defer s.LockUnlock()()
 
-	s.files[filePath] = body
+	s.files[filePath] = FileEntry{
+		Info: workspace.ObjectInfo{
+			ObjectType: "FILE",
+			Path:       filePath,
+			Language:   "SCALA",
```

Comment:
Setting `language` here does not match API behaviour. It's only set for notebooks: https://openapi.dev.databricks.com/api/workspace/workspace/import#language

FYI the import logic is more complex than this, because the service also makes a determination of whether something is a file or a notebook. We don't need to address that in this PR.

Reply:
Yes, I'm just following previous default which always returned "SCALA".